functions_lab.js

```diff
@@ -0,0 +1,418 @@
+// # Function Exercises
```

Comment:
I see some feedbacks on few questions as stated below. Always try to run your code with differing inputs before moving on to next question. Also it is super important to understand how return works when implemented inside a for loop. Try to re-solve some of questions and let me know if you have any questions. Keep it up!